packages/runner/webpack.config.ts

```diff
@@ -1,6 +1,17 @@
 import _ from 'lodash'
-import commonConfig, { HtmlWebpackPlugin } from '@packages/web-config/webpack.config.base'
+import getCommonConfig, { HtmlWebpackPlugin } from '@packages/web-config/webpack.config.base'
 import path from 'path'
+import webpack from 'webpack'
+
+const commonConfig = getCommonConfig()
+
+// @ts-ignore
+commonConfig.module.rules[1].use.options.plugins.push([require.resolve('babel-plugin-prismjs'), {
+  'languages': ['javascript'],
```

Comment:
why do we set this here but then also set this in `reporter` and use different `languages` options?

https://github.com/cypress-io/cypress/pull/3930/files#diff-79efa94cf0c17d7e9b81ba8d6f3fce37R3

Reply:
The `add-prism-transform.js` was added before switching to webpack and can be removed.